README.md

```diff
@@ -57,6 +57,57 @@ You probably have enabled "Secure Window" feature in SealNote settings.
 Disable it to be able to take screenshots or show window content in application
 switcher.
 
+Contributing
+--------------------------------
+
+Contributions are much appreciated. Please go through this section to understand how and what to contribute.
+
+### Tools of trade
+
+1. Java 1.7+
+1. Gradle 0.12.2
+2. Editor of your choice (Please do not check-in editor project files)
+
+### What to contribute
+
+SealNote is an application which focuses on security and simplicity.
+
+If you need ideas for contribution, please check currently open issues.
+
+If you want to add a new feature, please open a new issue. A good feature can be something which improves some workflow or security for a user.
+
+### Issue assignment
+
+Before you start working on an issue, please claim it so that there is no duplication of effort. We don't assign an issue until you have a passing pull-request.
```

Comment:
I think this section along with "What to contribute" section could be merged together, and put right after "Contributing" without any title. Its probably three most important lines, so should probably go up.